air/src/proof.rs

```diff
@@ -218,16 +201,3 @@ impl TryFrom<u8> for HashFunction {
         }
     }
 }
-
-impl FromStr for HashFunction {
```

Comment:
This is useful to allow the CLI to parse the arguments directly into the enum

Reply:
I'm not yet sure we need to expose hash function specifics to the CLI. I'd rather have more intent-based options there. For example, we could have a flag to specify whether we want a "recursion-friendly" proof or not, and based on that we'd choose which hash function to use.